src/request/caches/modes.rs

```diff
@@ -1,15 +1,125 @@
 use serde::{Deserialize, Serialize};
 
-const DEFAULT_CONCURRENCY_LEVEL: i32 = 1000;
-const DEFAULT_ACQUIRE_TIMEOUT: i32 = 15000;
+const DEFAULT_CONCURRENCY_LEVEL: i32 = 1_000;
+const DEFAULT_ACQUIRE_TIMEOUT: i32 = 15_000;
+const DEFAULT_STATE_TRANSFER_TIMEOUT: i32 = 60_000;
+const DEFAULT_REMOTE_TIMEOUT: i32 = 17_500;
 
-#[derive(Serialize, Deserialize)]
-pub struct LocalCache {
+#[derive(Eq, PartialEq, Debug, Serialize, Deserialize)]
+pub struct Local {
     locking: Locking,
     statistics: bool,
 }
 
-#[derive(Serialize, Deserialize)]
+impl Default for Local {
+    fn default() -> Self {
+        Self {
+            locking: Locking::default(),
+            statistics: true,
+        }
+    }
+}
+
+#[derive(Eq, PartialEq, Debug, Serialize, Deserialize)]
+pub struct Distributed {
+    mode: String,
+    #[serde(rename = "state-transfer")]
```

Comment:
This might also be the case that you want to tag the whole struct as serde's `kebab-case`.

Reply:
Fixed :+1: